data/src/main/java/com/thanht/data/util/JsonUtils.kt

```diff
@@ -0,0 +1,50 @@
+package com.thanht.data.util
+
+import android.content.Context
+import android.text.TextUtils
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStream
+import java.io.InputStreamReader
+
+object JsonUtils {
```

Comment:
Copy but should know about it clearly. when you decompile it is static `gSon` variable, it means with your code `gSon` never null